src/core/auth/components/LoginFormContainer.tsx

```diff
@@ -1,11 +1,40 @@
-import { memo } from 'react';
 import { LoginForm } from './LoginForm';
+import { ChangeEvent, useMemo } from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import { selectUserName, selectIsLoading, changeUserName, selectErrorMessage, changePassword, selectPassword, setIsLoading } from '../redux/authSlice';
 
 // eslint-disable-next-line react/display-name
-export const LoginFormContainer = memo(() => {
+
+export const LoginFormContainer = () => {
+  const dispatch = useDispatch();
+
+  const userName = useSelector(selectUserName, shallowEqual);
+  const isLoading = useSelector(selectIsLoading, shallowEqual);
+  const errorMessage = useSelector(selectErrorMessage, shallowEqual);
+  const password = useSelector(selectPassword, shallowEqual);
+
+  const isSubmitDisabled = useMemo(() => userName === '' || password === '', [userName, password]);
+
+  const handleChangeUserName = ({ target }: ChangeEvent<HTMLInputElement>) => {
+    dispatch(changeUserName(target.value));
+  };
+
+  const handleChangePassword = ({ target }: ChangeEvent<HTMLInputElement>) => {
+    dispatch(changePassword(target.value));
+  };
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  //@ts-ignore
+
+  const handleSubmit = () => {
+    dispatch(setIsLoading(true));
+  };
+
+  const loginFormProps = { handleChangeUserName, isSubmitDisabled, isLoading, handleChangePassword, handleSubmit, errorMessage, title: '[ta-rung]' };
```

Comment:
Looks like the title here is a redundant prop and can be removed

Reply:
Ok, I will remove title in the next commit!